### PR TITLE
[RT-67] Add basic implementation

### PR DESCRIPTION
**Pull request by MLGMAG**

### CODE REVIEW

File: src/main/java/com/webmuffins/rtsx/messenger/controller/MessageController.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import javax.annotation.Resource;
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/messages")
+public class MessageController {
+
+    @Resource
```
Comment (makrushinoff) [inline src/main/java/com/webmuffins/rtsx/messenger/controller/MessageController.java]: Remove this annotation

File: src/main/java/com/webmuffins/rtsx/messenger/converter/Converter.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.webmuffins.rtsx.messenger.converter;
+
+public interface Converter<SOURCE, TARGET> {
```
Comment (makrushinoff) [inline src/main/java/com/webmuffins/rtsx/messenger/converter/Converter.java]: Use single letter

File: src/main/java/com/webmuffins/rtsx/messenger/converter/MessageResponseMessageConverter.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        message.setMessageText(messageRequest.getMessageText());
+        populateCreationDate(message);
+        return message;
+    }
+
+    private void populateCreationDate(Message messageObj) {
+        DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern(dateTimeFormat);
+        LocalDateTime creationDate = LocalDateTime.now();
+        String formattedCreationDate = creationDate.format(dateFormat);
+        messageObj.setCreationDate(formattedCreationDate);
```
Comment (makrushinoff) [inline src/main/java/com/webmuffins/rtsx/messenger/converter/MessageResponseMessageConverter.java]: Use mapper style

File: src/main/resources/application.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+      authentication-database: admin
+      username: ${DB_USERNAME}
+      password: ${DB_PASSWORD}
+      database: ${DB_NAME}
+      port: ${DB_PORT}
+      host: ${DB_HOST}
+
+message:
+  creationDate:
+    pattern: 'dd/MM/yyyy HH:mm:ss.SSS'
```
Comment (makrushinoff) [inline src/main/resources/application.yaml]: use application.properties file

**Review by makrushinoff — CHANGES_REQUESTED** [review]

File: src/main/java/com/webmuffins/rtsx/messenger/converter/Converter.java (hunk 2 above)
Comment (MLGMAG) [reply src/main/java/com/webmuffins/rtsx/messenger/converter/Converter.java]: Done

File: src/main/java/com/webmuffins/rtsx/messenger/converter/MessageResponseMessageConverter.java (hunk 3 above)
Comment (MLGMAG) [reply src/main/java/com/webmuffins/rtsx/messenger/converter/MessageResponseMessageConverter.java]: Done

File: src/main/resources/application.yaml (hunk 4 above)
Comment (MLGMAG) [reply src/main/resources/application.yaml]: Done

**Review by makrushinoff — APPROVED** [review]

**Pull request merged**